sympy/functions/special/error_functions.py

```diff
@@ -1,15 +1,15 @@
 """ This module contains various functions that are special cases
     of incomplete gamma functions. It should probably be renamed. """
 
-from sympy.core import EulerGamma  # Must be imported from core, not core.numbers
```

Comment:
I don't know if the existing comment applies to `oo` or not.  Perhaps better to do the `oo` import on a new line?

Reply:
It would be better to just use `S.EulerGamma`.

Reply:
And `S.Infinity`.